samples/Azure.Analytics.Purview.Account/readme.md

```diff
@@ -6,6 +6,7 @@ namespace: Azure.Analytics.Purview.Account
 data-plane: true
 security: AADToken
 security-scopes:  https://purview.azure.net/.default
+single-top-level-client: false
```

Comment:
Normally it's a convention that the default value of not-defined property should be zero value (e.g., false, 0, null). It's a little bit weird that the default value is true.
It would be better if the property name can be changed to `multiple-top-level-clients`.

Reply:
Why this readme.md still has configuration `data-plane: true`? Shouldn't it be removed as we already make it default?

Reply:
Remove all data-plane: true